profile/README.md

```diff
@@ -1,11 +1,76 @@
+
+Nautilus strongly accepts the reality that open-source software becomes dominant in the long-term; that the open-collaboration and competition of ideas leads the best outcomes succeeding, and that open-source exemplifies this in a very practical and real way.
+
+While there can be temporary advantage, particularly for new project or company in the to protect their "secret sauce", or source code, from competing eyes; Nautilus has the luxury for that its business foals are aligned orthogonal from market direct market product competition — instead we positioning ourselves to be attractive for great talent: whereas more loving attention towards our projects, either competitive or not, will directly contribute to our success.
+
+The Block Finance Group, like everyone, shares the fundamental grounding belief to be good and kind is the fundament of fulfillment: profit must in monetary success in harmony with the betterment world.
+
+Nautilus places great importance of being fair, kind, reasonable, transparent, intelligent, and empathetic. We wish to foster a culture of personal growth, leading to co-independence: where teams and individuals act in self-responsibility and cooperation. Our corporate structures continually evolve through self-and-group reflection; leading to a greater fostering our ethos in all parts of our operations.
+
+## Projects
+
+Since 2023, the primary focus of Nautilus has been on the Torrust Project:
+
+### Torrust
+
+
+`Torrust` is a group of programs that allow for the better management of torrent collections. Nautilus made the commitment to strategically foster this project as it is confident that it is in close alignment with both the Ethos of the company, (and group), and the real company goal nearing itself to a very interesting natural talent pool.
+
+in particular, our ethos is can be expressed as a:
+
+- value of sharing and archival of information. (such as AI and experimental datasets)
+- interest of open networking systems, and standards.
+- indexing and protection of data for future generations.
+
+In addition, for more practical reasons:
+
+- rust is an interesting and increasingly popular programming language
+- there is large communities of passionate users of other torrent trackers and indexes
+
+#### Torrust Tracker
+
+A `Torrent Tracker` is a set of server services that allow for the torrent swam to match-make while optionally keeping useful statistics for better insight and choice making.
+
+The tracker connects peers who share the same torrent (infohash, i.e. a unique specifier for a torrent). A peer that is missing parts or all of the torrent data is then able to connect to other peers, and obtain the parts they are missing.
+
+The torrust tracker is unique as it is a modern implementation in the excellent rust programming language. We hope that this project will become well-used and provide a real benefit to many; and will attract a vibrant community around it's continued improvement and maintenance.
```

Comment:
Is there any other additional difference as to why our tracker is a better implementation? Rust only cannot be the sole reason I think. I would add if you and @josecelano agree that we have better:
- Documentation
- Code quality
- Dockerization
- Etc.

Reply:
In addition I think we should avoid using words such as "hope".
For me the way hope is used shows insecurity. I strongly believe that we have the right which stems from our work and dedication to express ourselves with confidence.